irksome/dirk_stepper.py

```diff
@@ -101,31 +50,28 @@ def getFormDIRK(F, butch, t, dt, u0, bcs=None):
     stage_F = replace(F, repl)
 
     bcnew = []
-    gblah = []
 
     # For the DIRK case, we need one new BC for each old one (rather
     # than one per stage), but we need a `Function` inside of each BC
     # and a rule for computing that function at each time for each
     # stage.
+    a_vals = numpy.array([MC.Constant(0) for i in range(num_stages)],
+                         dtype=object)
+    d_val = MC.Constant(1.0)
     for bc in bcs:
```

Comment:
It'd be nice if we could find a unified treatment for stage BC, and handle everything BC-related within bcs.py. But that could come as a separate PR.

Reply:
I was thinking a bit about this as I wrote the code here.  Might not be too difficult, but I agree that it'd be better to save it for a separate PR

Reply:
I agree that fixing it is good, but a separate issue.  BCS have been the bane of Irksome for most of its existence.  I’m thrilled with the progress y’all are making on this…